### PR TITLE
Update tests for creating variants

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I updated tests for creating variants

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5436 Update tests for creating variants](https://app.clickup.com/t/2549495/SALEOR-5436)

**Review by eCzerniak — CHANGES_REQUESTED** [review]
There is a problem with field SKU:
- should create variant visible on frontend
- should create variant for many channels
those tests are failing.
image

**Comment by karola312** [comment]
I added data-test-id attribute to sku input field

**Review by eCzerniak — APPROVED** [review]

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorSummary.tsx [hunk 1]
```diff
@@ -303,6 +303,7 @@ const ProductVariantCreatorSummary: React.FC<ProductVariantCreatorSummaryProps>
               <div className={classNames(classes.col, classes.colSku)}>
                 <TextField
                   name="sku"
+                  data-test-id="sku-input"
```
Comment (jwm0) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorSummary.tsx:306]: Can't we just get it by `cy.get('input[name="sku"]')`?

Comment (karola312) [reply src/products/components/ProductVariantCreatorPage/ProductVariantCreatorSummary.tsx:306]: Yes, I removed data test id

**Review by jwm0 — APPROVED** [review]

**Pull request merged**